test/integration/gcp/controls/forseti.rb

```diff
@@ -0,0 +1,136 @@
+# encoding: utf-8
+# copyright: 2018, The Authors
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+title 'Forseti Terraform GCP Test Suite'
+
+gcp_project_id = attribute('gcp_project_id')
+#gcp_org_id = attribute('gcp_org_id')
+
+control 'forseti-client-vm' do
+  impact 1.0
```

Comment:
Since we aren't using inspec for compliance, impact isn't terribly relevant for these tests. It doesn't hurt anything to have it here, but we don't get anything from it either.